usb-gadget/hal/file_contexts

```diff
@@ -0,0 +1,3 @@
+# usb role switch service
+/sys/class/typec  						u:object_r:sysfs_usb_typec:s0
```

Comment:
Please move this line to genfs_contexts:
genfscon sysfs /class/typec               u:object_r:sysfs_usb_typec:s0